### PR TITLE
NEWS: some nitpicking and bike-shedding

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -176,8 +174,8 @@ CHANGES WITH 235:
           enforced on every single IPv4 and IPv6 socket created by any process
           of the service unit, and apply to ingress as well as egress traffic.
 
-        * If CPUAccounting= or IPAccounting= is turned on for a unit a new,
-          recognizable log message is generated each time the unit is stopped,
+        * If CPUAccounting= or IPAccounting= is turned on for a unit a new
+          standarized log message is generated each time the unit is stopped,
```
Comment (lucaswerkmeister) [inline NEWS]: s/standarized/standar**d**ized/

File: NEWS [hunk 2]
```diff
@@ -12,22 +12,21 @@ CHANGES WITH 235:
...
           to on. If turned off kernel log messages will not be read by
-          systemd-journald and not be included in the logs. It also gained a
-          new setting LineMax= for configuring the maximum line length to allow
-          when converting STDOUT/STDERR log streams into individual log
-          records. The new default for this value is 48K, up from the previous
-          hardcoded 4K.
+          systemd-journald or included in the logs. It also gained a new
+          setting LineMax= for configuring the maximum line length in
+          STDOUT/STDERR log streams. The new default for this value is 48K, up
+          from the previous hardcoded 2048.
```
Comment (lucaswerkmeister) [inline NEWS]: Is the change of the documented old limit (4K → 2048) correct? (I tried looking at the old code, but couldn’t figure out where the old limit was applied.)

Comment (keszybz) [reply NEWS]:
```console
$ getconf LINE_MAX
2048
```

Comment (lucaswerkmeister) [reply NEWS]: Ah, thanks.

**Comment by keszybz** [comment]
Oops, I pushed the first commit by mistake along with the merge of #6952. So now this PR is just the rest + the typo fix.

**Comment by lucaswerkmeister** [comment]
> Also fix typo (by using a word that is a better git anyway.)

…a better **f**it? :)

**Pull request merged**